pennylane_ionq/api_client.py

```diff
@@ -482,7 +482,6 @@ class JobResult(Resource):
     """
 
     SUPPORTED_METHODS = ("GET",)
-    PATH = "v0/jobs/{job_id}/data"
```

Comment:
These parts appear to be legacy code that is never used in this plugin's workflow